back/news/templates/news/shotguns.html

```diff
@@ -23,124 +27,109 @@
+                </div><!--End of Box-->
+            {% empty %}
+                <!--Box-->
+                <div class="box small-box">
+                    <h5 class="section-title">Aucun shotgun à venir.</h5>
+                </div><!--End of Box-->
+            {% endfor %}
+
+            <h3 class="page-title">Mes Shotguns</h3>
+
+            <div class="centered-div mt-2">
                 {% if display_admin_button %}
-                    <br>
                     <a href="{% url 'news:shotguns_admin' %}" style="text-decoration: none"><button class="button blue-button">Administration des shotguns</button></a>
+                    <br>
                 {% endif %}
+                <a href="{% url 'news:new_shotgun' %}"><i class="fas fa-plus-circle fa-5x"></i></a>
             </div>
-        </div>
-    </div>
-    <script src="{% static 'js/sidenav.js' %}"></script>
-</body>
-</html>
+
+
+            {% for shotgun in user_shotguns %}
+                <!--Box-->
+                <div class="box small-box">
+                    <a class="text-not-underlined" href="{% url 'news:shotgun_detail' shotgun.id %}">
+                        <h5 class="section-title">
+                            <strong class="text-bold">{{shotgun.title}}</strong>
+                            <br>
+                            <strong class="text-thin">Organisé par {{shotgun.club.name}}</strong>
+                        </h5>
+                    </a>
+                </div><!--End of Box-->
+            {% empty %}
+                <!--Box-->
+                <div class="box small-box">
+                    <h5 class="section-title">Tu n'as participé à aucun shotgun.</h5>
+                </div><!--End of Box-->
+            {% endfor %}
+
+            <h3 class="page-title">Anciens Shotguns</h3>
+
+            <!--Row-->
+            <div class="row">
+
+                {%for shotgun in old_shotguns%}
+                    <!--Column-->
+                    <div class="col-clublg-6">
+                        <div class="box small-box">
+                            <a class="text-not-underlined" href="{% url 'news:shotgun_detail' shotgun.id %}">
+                                <h5 class="section-title">
+                                    <strong class="text-bold">{{shotgun.title}}</strong>
+                                    <br>
+                                    <strong class="text-thin">Organisé par {{shotgun.club.name}}</strong>
+                                </h5>
+                            </a>
+                        </div><!--End of Box-->
+                    </div><!--End of column-->
+                {% empty %}
+                    <!--Box-->
+                    <div class="box small-box mt-2">
+                        <h5 class="section-title">Aucun shotgun terminé.</h5>
+                    </div><!--End of Box-->
+                {% endfor %}
+
+
+            </div><!--End of Row-->
+
```

Comment:
```suggestion
<div class="centered-div mt-2">
                {% if display_admin_button %}
                    <a href="{% url 'news:shotguns_admin' %}" style="text-decoration: none"><button class="button blue-button">Administration des shotguns</button></a>
                    <br>
                {% endif %}
                <a href="{% url 'news:new_shotgun' %}"><i class="fas fa-plus-circle fa-5x"></i></a>
            </div>
```
Quelque chose dans ce style pour mettre les boutons d'administration en bas.